Makefile

```diff
@@ -42,9 +45,9 @@ clean: ## - Clean up build artifacts
 .PHONY: generate
 generate: ## - Generate schema models
 	@printf "${CMD_COLOR_ON} Installing module for go generate\n${CMD_COLOR_OFF}"
-	go install github.com/aleksmaus/generate/...
+	env GOBIN=${GOBIN} go install github.com/aleksmaus/generate/cmd/schema-generate@latest
```

Comment:
is it in the plan prior to 7.14 to remove alexmaus personal git directory as part of source?

Reply:
That would be wise.  @aleksmaus can handle that when he returns.  Also noticed a [tool](https://github.com/elastic/fleet-server/blob/master/dev-tools/common.bash#L43) in Andrew Kroh's personal space.  Not sure where that came from.

Reply:
This was a fork and heavily customized schema code generator for our specific needs.
FWIW, we pull bunch of third party dependencies from other people personal spaces ;-)

Can move it at some point into elastic org sure.